vivarium/core/process.py

```diff
@@ -153,7 +153,7 @@ def __getstate__(self) -> dict:
 
     def __setstate__(self, state: dict) -> None:
         """Initialize process with parameters"""
-        self.__init__(parameters=state)  # type: ignore
+        self.__init__(state)  # type: ignore
```

Comment:
can you call this `parameters`, instead of `state`?

Reply:
done!